pkg/daemon/daemon.go

```diff
@@ -185,6 +195,10 @@ func (dn *Daemon) nodeStateAddHandler(obj interface{}) {
 	}
 
 	if reqReboot {
+		if dn.noReboot {
```

Comment:
I think it would be better to invoke `rebootDisabled()` before executing the reboot. Since the SriovOperatorConfig CR may be changed after the config daemon is up.